app/controllers/home_controller.rb

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
 class HomeController < ApplicationController
-  def index; end
+  def index
+    men_cat_id = Category.find_by(name: 'Men')
```

Comment:
lepsze nazwy zmiennych, to nie jest `men_cat_id` tylko `men_category` (to nie jest `id`)
to byłoby ID
`men_cat_id = Category.find_by(name: 'Men').id`